### ISSUE TITLE
variants annotate -E option should take parameter

### ISSUE THREAD

**Issue by yanyangfeng** [griffithlab/regtools#60]
-I option probably not necessary
-E option should take a parameter which is basically the same as the corresponding run with -e, just tells you the minimum distance from the edge of the exon, to allow for separate runs

**Comment by gatoravi** [comment]
-E (-I) was meant to be a binary option to annotate a variant that falls within any part of the exon (intron). Would it be possible to do what you're suggesting with just the -e option?

**Comment by yanyangfeng** [comment]
Sorry, I should've explained this better. But Malachi was suggesting that it could be useful to separate variants into two separate batches: sort of the high-priority, edge of exon variants, and then the rest of the exonic variants (ensuring no overlap with the exonic variants close to the edge). If the user knows he wants both of these batches and wants to get them all in one run, I suppose it would work to just do -e <window size> -E and infer the parameter, but if we wanted to allow separate runs for these batches (perhaps even just to run jobs in parallel), we'd need to allow an argument to be passed to -E.

**Comment by malachig** [comment]
Yeah basically we want to be able to run regtools in two broad ways. 

(1) focus on the splice sites (exon edges).  Where we identify variants X bases into the exon, and Y bases into the intron.

(2) focus on the internal parts of exons.  But to avoid repeating effort above, look for variants that are anywhere within the exon *except* within X bases from the edge of the exon.  

If the -E option had a flank option that allowed you to reduce or expand the region of the exon being considered this would give us flexibility to run in various ways with various types of focus.
